eve_swagger/paths.py

```diff
@@ -16,6 +17,9 @@
 
 # TODO consider adding at least a 'schema' property to response objects
 # TODO take auth into consideration
+def _clear_regex(_str):
```

Comment:
Since this is likely to be only used in `paths()`, consider making it a closure of `paths` itself.